build.gradle

```diff
@@ -613,6 +621,23 @@ dependencies {
 
     // Bouncy Castle Libaray
     compile 'org.bouncycastle:bcprov-jdk15on:1.48'
+
+    // Gobblin
+    compile ('com.linkedin.gobblin:gobblin-runtime:0.10.0') {
+        exclude group: 'org.apache.hive', module: 'hive-exec'
```

Comment:
These exclude clauses are repetitive. There's probably a way to define them once and then reuse them as a Groovy closure or something similar. Ideally, all three exclusions should be defined as a single exclusion closure.